staging/src/k8s.io/apimachinery/pkg/apimachinery/types.go

```diff
@@ -26,9 +26,6 @@ import (
 
 // GroupMeta stores the metadata of a group.
 type GroupMeta struct {
-	// GroupVersion represents the preferred version of the group.
-	GroupVersion schema.GroupVersion
-
 	// GroupVersions is Group + all versions in that group.
```

Comment:
// GroupVersions all versions of the specified group, starting with the prefered one. This must not be empty.